ci_js/doc/index.rst

```diff
@@ -0,0 +1,60 @@
+========
+ CI JS
+========
+
+
+Configuration
+=============
+
+
+Usage
+=====
+
+Создание нового тура
+--------------------
+
+* Открыть в верхнем меню ``[[ CI JS ]]``
+* В боковом меню во вкладке ``CI JS`` выбрать кнопку ``Tours``
```

Comment:
Usually this is replaced by one line:
Open menu ``[[ CI JS ]] >> CI JS >> Tours``